Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:20 as Base
+WORKDIR /app
+COPY package*.json  ./
```

Comment:
No need for `package*.json`
you can simply mention up the package.json , what I think so

Reply:
package-lock.json - will be not included then..
That was fine.